CMakeLists.txt

```diff
@@ -265,7 +309,7 @@ if(WIN32)
     endif()
 elseif(APPLE)
     set_source_files_properties(images/icon.icns PROPERTIES MACOSX_PACKAGE_LOCATION "Resources")
-    install(TARGETS hdrmerge BUNDLE DESTINATION "/Applications")
+    install(TARGETS hdrmerge BUNDLE DESTINATION "~/hdrmerge/build/release")
```

Comment:
There are several issues here.
* Is `~` safe? In Linux one generally uses `$HOME` instead.
* The build type could be something other than release, so I'd replace that with `${CMAKE_BUILD_TYPE}`
* Instead of using that at all, better to use `DESTINATION ${CMAKE_INSTALL_BINDIR}`, and then to set that variable when running cmake:
```bash
cmake \
    -DCMAKE_CXX_FLAGS="-std=c++11 -Wno-deprecated-declarations -Wno-unused-result" \
    -DCMAKE_INSTALL_BINDIR:STRING="$HOME/programs/hdrmerge-release" \
    -DCMAKE_BUILD_TYPE="Release"  \
    "$HOME/programs/code-hdrmerge"
```
See https://github.com/jcelaya/hdrmerge/blob/master/build-hdrmerge#L126

Reply:
Thanks, revised to incorporate changes.